functions/Get-DbaAgentSchedule.ps1

```diff
@@ -0,0 +1,78 @@
+FUNCTION Get-DbaAgentSchedule
```

Comment:
Ultimately, what I'm driving at is being able to create a new schedule without having to specify which job it's associated with. I've found that this can be done by specifying the `JobServer` object as the parent object when creating a new schedule, rather than having to specify a Job object.

In `[New-DbaAgentSchedule](https://github.com/sqlcollaborative/dbatools/blob/master/functions/New-DbaAgentSchedule.ps1#L7)` it specifies that creating a schedule does not attach it to the job - this is actually not the case; creating a new schedule using this function does associate the schedule with the referenced job.

I see two options here - either make the `Job` parameter optional (and if it isn't specified then create the schedule with the `JobServer` as the parent), or split the functions into `New-DbaAgentSchedule` and `New-DbaAgentJobSchedule` , for example.

What are your thoughts? I'm conscious that this conversation is getting a bit long - feel fee to move it into a Slack channel if you prefer?